components/LocationList.js

```diff
@@ -0,0 +1,28 @@
+import Card from "./Card";
+import styled from "styled-components";
+
+const ListElement = styled.ul`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-left: 0;
+`;
+
+export default function LocationList() {
+  return (
+    <ListElement>
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
+      <Card />
```

Comment:
fine as a starting point, but as soon as possible, you should make this part of the code dynamic :)

Reply:
Yes, that is part of the next user story! This list is only a placeholder for the beginning.